docs/README.md

```diff
@@ -74,6 +74,13 @@ Follow these steps if this is the first time you are setting up to work on the d
 >
 > Only now do `git` commands start to work normally. Now, too (but not before) `npm ci` works properly.
 
+Note:
+For Catalina, an additional step maybe necessary to make sure xcode-select is pointing to the Xcode Developer directory using a command like this:
+
+```
+xcode-select -s /Applications/Xcode.app/Contents/Developer
```

Comment:
Per its manpage, `xcode-select --switch` should point just to the version of Xcode you want it to use, and should be run with `sudo`:

> Sets the active developer directory to the given path, for example **/Applications/Xcode-beta.app**. This command must be run with superuser permissions [...]

```suggestion
sudo xcode-select --switch /Applications/Xcode.app
```

Reply:
And if someone installed the command-line tools standalone, they live in /Library/Developer/CommandLineTools...

I think we need to revisit this whole section with a fresh macOS install, and see what happens. You don't happen to have a fresh machine, do you? I don't, right now. :)

Reply:
I do not have a fresh machine at this point as well. Will review the other changes you suggested.